source/_integrations/zwave_js.markdown

```diff
@@ -1008,32 +1008,48 @@ You can switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on
 
 ### How to switch from Z-Wave JS to the Z-Wave JS UI add-on?
 
-You can switch from the official **Z-Wave JS** add-on to the **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
+You can switch from the official **Z-Wave JS** add-on to the community **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
 
-Switching does not require renaming your devices.
+All of your HA devices and entities will remain unchanged if you follow these steps:
 
 1. Note your network security keys from the official add-on.
+   - In your browser, open **Settings** -> **Add-ons** -> **Z-Wave JS** -> **Configuration**.  ([direct link](http://homeassistant.local:8123/hassio/addon/core_zwave_js/config))
+   - From the three-dot {% icon "mdi:dots-vertical" %} menu, pick **Edit in YAML**.
+   - You should see about 12 lines of YAML, including items like `device: xxx` and `s2_access_control_key: xxx`.  Select all and copy somewhere safe.  You will need them later.
 
-2. Install and start the **Z-Wave JS UI** add-on.
+2. Install and start the community **Z-Wave JS UI** add-on.
+   - In your browser, open **Settings** -> **Add-ons** -> **Add-on Store** -> **Z-Wave JS UI**.  ([direct link](http://homeassistant.local:8123/hassio/addon/a0d7b954_zwavejs2mqtt/info))
+   - Click **Install**, then **Start**.
```

Comment:
We normally use the word "press" instead of "click", as we don't know that the user is using a mouse.

Reply:
:smile:  we usually use "press" when [referring to physical buttons](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold), and we [prefer select over click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click) because they might not use a mouse, like you said.